examples/citizenship.jsonld

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/citizenship/v2"
+  ],
+  "id": "https://issuer.oidp.uscis.example.gov/credentials/83627465",
+  "type": ["VerifiableCredential", "CertificateOfCitizenshipCredential"],
+  "name": "Certificate of Citizenship",
+  "description": "Example Country Certificate of Citizenship.",
+  "issuer": "did:example:28394728934792387",
+  "issuanceDate": "2019-12-03T12:19:52Z",
+  "credentialSubject": {
+    "id": "did:example:b34ca6cd37bbf23",
+    "type": ["Person", "Citizen"],
+    "givenName": "JOHN",
+    "additionalName": "JACOB",
+    "familyName": "SMITH",
+    "image": "data:image/png;base64,iVBORw0KGgo...kJggg==",
+    "gender": "Male",
+    "residentSince": "2015-01-01",
+    "commuterClassification": "C1",
+    "birthCountry": "Bahamas",
+    "birthDate": "1958-07-17",
+    "height": {
+      "type": "QuantitativeValue",
+      "unitCode": "INH",
+      "value": 71
+    },
+    "residence": {
+      "type": "PostalAddress",
+      "addressLocality": "Boston",
+      "addressRegion": "Massachusetts",
+      "addressCountry": "US"
+    },
+    "formerNationality": "BS",
+    "maritalStatus": "married",
+    "marriageCertificateNumber": "A123B",
+    "marriageLocation": {
+      "type": "PostalAddress",
+      "addressLocality": "Nassau",
+      "addressRegion": "New Providence",
+      "addressCountry": "BS"
+    },
+    "certificateOfCitizenship": {
+      "type": "PermanentResidentCard",
+      "identifier": "83627465",
+      "cisRegistrationNumber": "999-999-999",
+      "filingLocation": {
+        "type": "PostalAddress",
+        "addressLocality": "Washington, D.C.",
+        "addressRegion": "District of Columbia",
+        "addressCountry": "US"
+      }
+    }
+  },
+  "proof": {
+    "type": "Ed25519Signature2018",
+    "created": "2020-01-30T03:32:15Z",
+    "jws": "eyJhbGciOiJFZERTQSIsI...wRG2fNmAx60Vi4Ag",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:example:28394728934792387#keys-7f83he7s8"
+  }
+}
+ 
```

Comment:
```suggestion
```

Reply:
Wait...I thought you _wanted_ trailing line endings... 🤦

Reply:
Yeah, the final real char and then a newline.  This has a `}`, newline, then a space:
```
$ curl -s https://raw.githubusercontent.com/w3c-ccg/citizenship-vocab/3f3e9a179e4a6889fe331174caf81c67bdade77a/examples/citizenship.jsonld | hd | tail -2
000007c0  73 38 22 0a 20 20 7d 0a  7d 0a 20                 |s8".  }.}. |
000007cb
```

Reply:
This is why not having _any_ whitespace at the end of a file is so nice. 😁 No guesswork. 😉